modules/gltf/src/gltf/create-gltf-model.ts

```diff
@@ -48,7 +49,7 @@ const vs = `
     #endif
 
     pbr_setPositionNormalTangentUV(positions, _NORMAL, _TANGENT, _TEXCOORD_0);
-    gl_Position = u_MVPMatrix * positions;
+    gl_Position = proj.u_MVPMatrix * positions;
```

Comment:
Nit: Is `proj` from the project module? It seems to have a different abbreviation than the module name? Is such an exception worth it?

Reply:
I was meaning to come back and change that, thanks for the reminder